goat_launch/params/locomotor_params.yaml

```diff
@@ -0,0 +1,29 @@
+robot_base_frame: base_link
```

Comment:
there should probably be global_planner_class and local_planner_class variables here